wagtail/users/templates/wagtailusers/groups/includes/page_permissions_formset.html

```diff
@@ -16,9 +16,9 @@
     <table class="listing">
         <col />
         {% for i in formset.permission_types %}
-            <col width="15%" />
+            <col width="5%" />
```

Comment:
Same here, we could probably use a higher value as there are only 3 permission types, but I decided to use `5%` for consistency.